internal/util/util.go

```diff
@@ -233,18 +233,6 @@ func IsNonWordChar(r rune) bool {
 	return !IsWordChar(r)
 }
 
-// IsUpperWordChar returns whether or not a rune is an 'upper word character'
-// Upper word characters are defined as numbers, upper-case letters or sub-word delimiters
-func IsUpperWordChar(r rune) bool {
-	return IsUpperAlphanumeric(r) || IsSubwordDelimiter(r)
-}
-
-// IsLowerWordChar returns whether or not a rune is a 'lower word character'
-// Lower word characters are defined as numbers, lower-case letters or sub-word delimiters
-func IsLowerWordChar(r rune) bool {
-	return IsLowerAlphanumeric(r) || IsSubwordDelimiter(r)
-}
```

Comment:
These were added not so long ago in #2665. @masmu do you remember why exactly did you add them?

Reply:
No specific reason as far as I recall.
Probably they were added in the early stages of the implementation and became obsolete during finalization. It has probably slipped through that they are not actually in use.

Reply:
I see. So I guess it's ok to remove these.

Reply:
Yep. Thanks for double checking.